index.php

```diff
@@ -0,0 +1,26 @@
+<?php
+	header('Content-Type: text/html; charset=utf-8');
+	require_once 'initialize.php';
+
+	$getData = [
+		'test'      => $_GET['test'] ?? 0,
```

Comment:
Implementation should not know anything about tests. Tests should be completely separated.